Sources/ProjectSpec/XCProjExtensions.swift

```diff
@@ -54,17 +54,15 @@ extension PBXProductType {
     }
 
     /// Function to determine when a dependendency should be embedded into the target
-    public func shouldEmbed(_ dependencyType: PBXProductType) -> Bool {
-        switch dependencyType {
-        case .staticLibrary, .staticFramework:
-            // Some dependendencies should not be embed, independently of the target type
+    public func shouldEmbed(_ dependencyTarget: Target) -> Bool {
+        switch dependencyTarget.defaultLinkage {
+        case .static:
             return false
-
         default:
```

Comment:
Now that we don't have to enumerate the full type, can we make the cases explicit instead of using default

Reply:
Replaced `default` with explicit cases. I'm not sure if this reads any better or if I should do something like
```
if dependencyTarget.defaultLinkage == .static {
    // Static dependencies should never embed
    return false
} else if isApp {
    // If target is an app, all dependencies should be embed (unless they're static)
    return true
} else if isTest, [.framework, .bundle].contains(dependencyTarget.type) {
    // If target is test, some dependencies should be embed (depending on their type)
    return true
} else {
    // If none of the above, do not embed the dependency
    return false
}
```